velox/dwio/common/ScanSpec.cpp

```diff
@@ -316,6 +316,9 @@ bool testFilter(
     return true;
   }
   if (type->isDecimal()) {
+    // The min and max value in the metadata for decimal type in Parquet can be
+    // stored in different physical types, including int32, int64 and
+    // fixed_len_byte_array. The loading of them is not supported in Metadata.
```

Comment:
I took a look at the support of row group skip for Parquet decimal and am glad to follow up in a separate PR if you think it is needed.